pkg/processor/test/suite/suite.go

```diff
@@ -109,21 +105,28 @@ func (suite *TestSuite) SetupTest() {
 
 // BlastHTTP is a stress test suite
 func (suite *TestSuite) BlastHTTP(configuration BlastConfiguration) {
+	var totalResults vegeta.Metrics
 
 	// get createFunctionOptions from given blastConfiguration
 	createFunctionOptions, err := suite.blastConfigurationToDeployOptions(&configuration)
 	suite.Require().NoError(err)
 
 	// deploy the function
-	_, err = suite.Platform.CreateFunction(createFunctionOptions)
-	suite.Require().NoError(err)
+	suite.deployFunction(createFunctionOptions, func(deployResult *platform.CreateFunctionResult) bool {
 
-	// blast the function
-	totalResults, err := suite.blastFunction(&configuration)
-	suite.Require().NoError(err)
+		// give the container time to stabilize before we blast him
+		time.Sleep(5 * time.Second)
```

Comment:
🤨 
Magic interval?
This looks like it will be env/machine dependent - is there something functional to wait for here instead of the heuristic?

Reply:
fixed, moved to a function where it probes the function until it responds 200 once